itests/eth_api_test.go

```diff
@@ -396,42 +399,102 @@ func TestNetVersion(t *testing.T) {
 func TestEthBlockNumberAliases(t *testing.T) {
 	blockTime := 2 * time.Millisecond
 	kit.QuietMiningLogs()
-	client, _, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.ThroughRPC())
-	ens.InterconnectAll().BeginMining(blockTime)
-	ens.Start()
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-	defer cancel()
+	testCases := []struct {
+		name                   string
+		f3Enabled              bool
+		f3DelayEpochs          int
+		expectedSafeEpoch      int
+		expectedFinalizedEpoch int
+	}{
+		{
+			name:                   "f3 disabled",
+			f3Enabled:              false,
+			expectedSafeEpoch:      31,
+			expectedFinalizedEpoch: 901,
```

Comment:
this is the baseline that we want to keep stable; it's the behaviour of the original test so we're essentially asserting that without f3 involved we're replicating the existing behaviour and not changing anything